8.7.0-alpha4 Release Notes

conceptualshark opened the pull request:
## Description

Includes notes or placeholders for alpha4 [labeled epics](https://github.com/orgs/camunda/projects/9/views/31).

**This will need to be backported to the new 8.7 branch once the 8.7 and 8.8 versions have been cut.** 

## When should this change go live?

<!-- PRs merged go to stage.docs.camunda.io first and must be manually released to docs.camunda.io. -->

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**. (add `bug` or `support` label)
- [ ] This is already available but undocumented and should be released within a week. (add `available & undocumented` label)
- [ ] This is on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (create draft PR and/or add `hold` label)
- [x] This is part of a scheduled **alpha or minor**. (add alpha or minor label)
- [ ] There is **no urgency** with this change (add `low prio` label)

## PR Checklist

<!-- Camunda maintains 18 months of minor versions. Backporting your change to multiple versions is common. -->

- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).
- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.

<!-- UNCOMMENT THIS SECTION IF APPLICABLE. Adding or removing pages requires extra steps.
- [ ] I included my new page in the sidebar file(s).
- [ ] I added a redirect for a renamed or deleted page to the .htaccess file.
-->

- [x] I added a DRI, team, or delegate as a reviewer for technical accuracy and grammar/style:
  - [ ] [Engineering team review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process)
  - [x] [Technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process) via `@camunda/tech-writers` unless working with an embedded writer.

<!-- UNCOMMENT THIS SECTION IF APPLICABLE. Changes to **docs infra**, including updates to workflows and adding new npm packages, must be first discussed via issue or #ask-c8-documentation and linked for context.
- [ ] My changes require a [docs infrastructure review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process). (add `dx` label) -->

mesellings commented on docs/reference/release-notes/870.md:
```diff
@@ -19,6 +19,24 @@ These release notes identify the new features included in 8.7, including [alpha
+| :--------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :-------------------------------------------------------------------------------------------- |
+| 11 February 2025 | <ul><li>[ Camunda 8 core ](https://github.com/camunda/camunda/releases/tag/8.7.0-alpha4)</li><li>[ Connectors ](https://github.com/camunda/connectors/releases/tag/8.7.0-alpha4)</li></ul> | <!-- [Release blog](https://camunda.com/blog/2025/02/camunda-alpha-release-february-2025) --> |
+
+### BPMN copilot <span class="badge badge--long" title="This feature affects SaaS">SaaS</span>
```
```suggestion
### BPMN Copilot <span class="badge badge--long" title="This feature affects SaaS">SaaS</span>
```
Something to check with @christinaausley - I think there were some conversations around whether this should be capitalized?

akeller replied:
https://camunda.slack.com/archives/C047D95HBE2/p1738762354633159

Lowercase copilot -> BPMN copilot

conceptualshark replied:
After this conversation, it landed on "Copilot": https://github.com/camunda/camunda-docs/pull/4735#issuecomment-2647837688

akeller replied:
TY! Clearly a great add for the WCoE

mesellings commented on docs/reference/release-notes/870.md:
```diff
@@ -19,6 +19,24 @@ These release notes identify the new features included in 8.7, including [alpha
+
+### Ad-hoc subprocesses
+
+The new Camunda version supports a new BPMN element: [the ad-hoc subprocess](/components/modeler/bpmn/ad-hoc/ad-hoc.md). This new kind of subprocess allows more flexible process flows with a compact visual representation. It is the first step towards dynamic processes and execution of ad-hoc activities.
```
```suggestion
A new [ad-hoc subprocess](/components/modeler/bpmn/ad-hoc/ad-hoc.md) BPMN element is supported. This new kind of subprocess allows more flexible process flows with a compact visual representation. It is the first step towards dynamic processes and execution of ad-hoc activities.
```

mesellings commented on docs/reference/release-notes/870.md:
```diff
@@ -19,6 +19,24 @@ These release notes identify the new features included in 8.7, including [alpha
+
+### BPMN copilot <span class="badge badge--long" title="This feature affects SaaS">SaaS</span>
+
+Thanks to Camunda's integrated BPMN copilot for SaaS, anybody can go from 0 to 80% of a process diagram in minutes. Users can generate process diagrams from natural language descriptions, then collaborate on it with their colleagues. The text interface means that even BPMN novices can make meaningful, accurate diagrams.
```
I guess we could do with adding some more technical detail here at a later date, but no rush for now - this just reads as very much a PM description, without any 'useful' info on the actual feature.

conceptualshark replied:
It's not terribly more descriptive, but I updated this to be less marketing-copy, and added in the new documentation link. 👍

mesellings replied:
It's better, nice job - adding the docs link helps a lot 🚀

mesellings reviewed (approved):
Lgtm! Just a few rewording suggestions as normal 😊 🚀

mesellings reviewed (approved):
Lgtm! 🚀

sbuettner commented:
> Spring SDK and Camunda REST API Migration
The Connectors experience is enhanced with the migration from the Spring Zeebe to the Camunda REST API, and the removal of dependency on the Operate client.

We can remove this for 8.7 as the API migration will only happen in 8.8 as the APIs are not available in 8.7

conceptualshark commented:
To help organize changes, I've added the API migration back to this PR, but removed it and added the appropriate messaging and notes to the 8.8-alpha1 PR: https://github.com/camunda/camunda-docs/pull/4945

akeller reviewed (approved):
Based on what I see in the PDP, this looks correct. Please follow up with Document Handling Capability team, but based on their updates it looks like no work was delivered with alpha4 and instead shifted to 8.8.0-alpha1. I don't view this as blocking, so this is approved!

conceptualshark commented:
Document handling confirmed as needing no release notes, so merging this ahead of the versioning this afternoon. 👍 Will follow up with the correct release blog link and any other changes.
